internal/pkg/state/registry/registry.go

```diff
@@ -169,6 +172,22 @@ func (s *Registry) ConfigRowsFrom(config ConfigKey) (rows []*ConfigRowState) {
 	return rows
 }
 
+func (s *Registry) GetPath(key Key) (PathInProject, bool) {
+	objectState, found := s.Get(key)
+	if !found {
+		return PathInProject{}, false
+	}
+	return objectState.GetPathInProject(), true
+}
+
+func (s *Registry) GetByPath(path string) (ObjectState, bool) {
+	key, found := s.namingRegistry.KeyByPath(path)
+	if !found {
+		return nil, false
+	}
+	return s.Get(key)
+}
+
```

Comment:
Pridane 2 nove metody, na ziskanie cesty ku objektu ... a naopak, na ziskanie objektu podla cesty.
Na to si uz tiez narazil, ze ako sa da ziskat objekt podla path, ... tak teraz to bude o dost jednoduchsie.